components/shooter.py

```diff
@@ -53,22 +53,19 @@ def setup(self):
                 ctre.StatusFrameEnhanced.Status_1_General, 250, 10
             )
 
-    def execute(self):
-        self.right_motor.set(
-            ctre.ControlMode.Velocity,
-            self.motor_speed * self.RPS_TO_CTRE_UNITS,
-        )
-        self.left_motor.set(
-            ctre.ControlMode.Velocity,
-            self.motor_speed * self.RPS_TO_CTRE_UNITS,
-        )
+    def execute(self) -> None:
+        # XXX: mypy type inference bug?
+        speed_rps: float = self.motor_speed
```

Comment:
I'm not sure why mypy can't figure out what the inner type is, but it's happy when I do this :shrug: